packages/flutter/test/cupertino/tableview_test.dart

```diff
@@ -0,0 +1,267 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  // The viewport size comes from the test system. We explicitly declare it
+  // here so that if a different viewport size is used we can change it for all
+  // tests in the suite.
+  const Size viewportSize = Size(800.0, 600.0);
+
+  group('Plain CupertinoTableView', () {
+    testWidgets('CupertinoTableView lays out prebuilt cells vertically like a list', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        const CupertinoApp(
+          home: CupertinoTableView.plain(
+            plainChildren: <Widget>[
+              SizedBox(key: Key('cell1'), width: double.infinity, height: 100.0),
+              SizedBox(key: Key('cell2'), width: double.infinity, height: 100.0),
+              SizedBox(key: Key('cell3'), width: double.infinity, height: 100.0),
+            ],
+          ),
+        ),
+      );
+
+      // Ensure that each cell in the list is laid out as wide as the viewport
+      // and with a top-edge y-value that corresponds to the cells being stacked
+      // vertically.
+      final Offset box1TopRight = tester.getTopRight(find.byKey(const Key('cell1')));
+      expect(box1TopRight, Size(viewportSize.width, 0.0));
+
+      final Offset box2TopRight = tester.getTopRight(find.byKey(const Key('cell2')));
+      expect(box2TopRight.dy, Size(viewportSize.width, 100.0));
```

Comment:
.dy is a double right?

Reply:
Yeah, I'm not sure what happened to some of these expectations.  Something got out of whack.